draft-ietf-dnsop-dns-capture-format-00.md

```diff
@@ -1345,28 +1346,47 @@ completeness were also compared to JSON.
   a pre-defined schema. The schema itself is always included in the
   data file. Data can therefore be stored untagged, for a smaller
   serialisation size, and be written and read by an Avro library.
+
+    * Avro libraries are available for C, C++, C#, Java, Python, Ruby
```

Comment:
We should add a 'at the time of writing' disclaimer to this list and the one ProtfBuf one